__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const app = require('../app')
 const db = require('../db/connection')
 const seed = require('../db/seeds/seed')
 const testData = require('../db/data/development-data/index')
+require('jest-sorted'); 
```

Comment:
I don't think this is necessary - you're not referencing it anywhere in your code as its not actually saved to a variable, maybe a rogue auto-import? 
Either way - it can be removed.